js/majors.js

```diff
@@ -2728,17 +2728,17 @@ window.majors = {
     ]
   },
 
-  m22: {
-    name: "22 -- Nuclear Science & Engineering (as of Sep '17)",
+  m22_old: {
```

Comment:
You'll have to keep this as `m22`; otherwise you'll break the existing roads which rely on this key. 

Alternatively, you can rename this one to `m22_old` and the new one to `m22`; there just needs to still be `m22` in here.

Reply:
Gotcha, I'll fix that right now. In my update to the pull request I'll also take out the NEET stuff for now, and put it in for a new pull request on March 15th. I'll keep the NEET functionality in there with a placeholder, so the students know it's coming.

Reply:
Also, you can merge this pull request now. I'll do a different one for March 15th with NEET requirements. That will get the Course 22 requirements all correct as soon as we can.

Reply:
Will do, I can close this pull request and put in one now for just the m22 changes. Then I'll do another one on March 15th for just the NEET threads and the additional CourseRoad functionality.